hail_search/web_app.py

```diff
@@ -29,12 +29,13 @@ async def status(request: web.Request) -> web.Response:
     return web.json_response({'success': True})
 
 
-def init_web_app():
+async def init_web_app():
+    hl.init(idempotent=True)
+    load_globals()
```

Comment:
not  an issue, but curious why its better to load globals before app initialization?

Reply:
I moved the line so that "hail" code was being executed together, but other than that no reason 🤷